### PR TITLE
Added Badges

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
Badges -
Badges on icons
Badges on Avatar

### CODE REVIEW

File: components/Alert/badge/badge.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+          <ul>
+            <li><a href="">Introduction</a></li>
+            <li><a href="">Colors</a></li>
+            <li><a href="">Typography</a></li>
+            <li><a href="">Conclusion</a></li>
+          
+          
+        </ul>
+          <h3>Components</h3>
+          <div class="divider"></div>
```
Comment (harshitadaswani) [inline components/Alert/badge/badge.html:38]: don't use empty div. Instead use class divider in above element

File: components/Alert/badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+          <li><a class="navigation__list--item" href="#">Docs</a></li>
+          <li><a class="navigation__list--item" href="#">Dark</a></li>
+        </ul>
+      </nav>
+      <div class="content">
+  
+        <div class="sidebar">
+  
+          <h3>Get Started</h3>
+          <div class="divider"></div>
```
Comment (harshitadaswani) [inline components/Alert/badge/badge.html:28]: same here

File: components/Alert/badge/badge.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  
+        </div>
+  
+        <div class="main--content">
+
+            <h1 class="heading">Badge</h1>
+  
+            <p>Badges are small circles, positioned either at top-right or bottom-right of the parent component. Badge can be used to display numbers, online / offline status, depending on where they are used .</p>
+  
+              <div class="divider"></div>
```
Comment (harshitadaswani) [inline components/Alert/badge/badge.html:58]: again

**Review by harshitadaswani — APPROVED** [review]
use prettier for better code readability and remove empty divs

File: components/Alert/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
+
+  .icons {
+    height: 5rem;
+  }
+  
+  .number {
+    position: absolute;
+    background-color: var(--primary);
+    color: var(--white);
+    padding: .5rem;
```
Comment (Bhakti199) [inline components/Alert/badge/badge.css:10]: .5 rem = 8px , use px for less than equal to 10px

File: components/Alert/badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  .showcase {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+        -ms-flex-pack: center;
+            justify-content: center;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
```
Comment (Bhakti199) [inline components/Alert/badge/badge.css:31]: use proper structure

**Review by Bhakti199 — APPROVED** [review]
everything looks cool, good to merge

File: components/Alert/badge/badge.css (hunk 4 above)
Comment (SuyashTiwariCoder) [reply components/Alert/badge/badge.css:10]: ok thanks

**Pull request merged**